firefox-ios/firefox-ios-tests/Tests/XCUITests/TopTabsTest.swift

```diff
@@ -397,6 +397,80 @@ class TopTabsTest: BaseTestCase {
         addTabsAndUndoCloseTabAction(isPrivate: true, nrOfTabs: 4)
     }
 
+    // https://testrail.stage.mozaws.net/index.php?/cases/view/2306868
```

Comment:
Just to double check that you change the automation status to completed once this PR lands right? (I think I have asked this many times already 🙈 )